katharsis-core/src/main/java/io/katharsis/jackson/serializer/ContainerSerializer.java

```diff
@@ -23,6 +23,7 @@
 import io.katharsis.response.Container;
 import io.katharsis.response.ContainerType;
 import io.katharsis.response.DataLinksContainer;
+import io.katharsis.response.LinkageContainer;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'io.katharsis.response.LinkageContainer'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUselessImportCheck)